package com.sapient.day4.exceptions;

public class ExceptionsDemo2 {

	public static void main(String[] args) {
		
		try {
			throw new Exception("Book not found");
		} catch (Exception e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}
		

	}

}
